sql/testdata/drop_table

```diff
@@ -0,0 +1,30 @@
+statement ok
+CREATE TABLE a (id INT PRIMARY KEY)
+
+statement ok
+CREATE TABLE b (id INT PRIMARY KEY)
+
+query TT colnames
+SHOW TABLES FROM test
+----
+Table
+a
+b
+
+statement ok
+DROP TABLE a
+
+query T colnames
+SHOW TABLES FROM test
+----
+Table
+b
+
```

Comment:
On Thu, Aug 6, 2015 at 11:11 AM XisiHuang <notifications@github.com> wrote:

> In sql/testdata/drop_table
> <https://github.com/cockroachdb/cockroach/pull/1935#discussion_r36425700>:
>
> > +query TT colnames
> > +SHOW TABLES FROM test
> > +----
> > +Table
> > +a
> > +b
> > +
> > +statement ok
> > +DROP TABLE a
> > +
> > +query T colnames
> > +SHOW TABLES FROM test
> > +----
> > +Table
> > +b
> > +
>
> @vivekmenezes <https://github.com/vivekmenezes> if i understand
> correctly, when the table is recreated with same name, it will still get a
> different table id.
> So even if the new table is empty, we still don't know the old table data
> is deleted.
> That's why I created another drop_test file and used the KV interface to
> test if the old data is deleted.
> I will do it as you said to be safer.
>
Thats true. Anyway, do add the test to also ensure that the ID doesn't get
recirculated. Thanks



> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1935/files#r36425700>.
>